aodndata/moorings/products_handler.py

```diff
@@ -60,6 +60,26 @@ def _read_manifest(self):
                 "manifest file '{self.input_file}' missing information (site_code, variables)".format(self=self)
             )
 
+    def _get_wfs_features(self, filter_list, propertyname='*'):
```

Comment:
I tend to avoid using "internal" names to functions when they don't change state of the class - the function apparently only extract a piece of information without changing any state. Hence, it should be "public" and without the leading ```_```.